### ISSUE TITLE
Please make this product cross-platform. It does not run on Windows right now.

### ISSUE THREAD

**Issue by metaperl** [pynecone-io/pynecone#51]
Looking through [a config file](https://github.com/pynecone-io/pynecone-examples/blob/main/dalle/pcconfig.py) we see this:

```python
config = pc.Config(
    app_name="dalle",
    bun_path="$HOME/.bun/bin/bun",
    db_url="sqlite:///pynecone.db",
    env=pc.Env.DEV,
)
```

This will not run on windows. There may be other unix-isms throughout this product. Please make this product platform-neutral.

**Comment by Alek99** [comment]
Thanks will get on this! Definitely is going to be a platform neutral platform

**Comment by picklelo** [comment]
We're using `bun` for JS package management, and it doesn't support Windows currently. So we may either need to wait for `bun` to be compatible, or do some workaround on Windows systems.

**Comment by picklelo** [comment]
This will be available on release v0.1.9

**Comment by picklelo** [comment]
Please create a new issue if this is not solved for you in v0.1.9
